### ISSUE TITLE
[FR] Integration with sklearn-evaluation for plots

### ISSUE THREAD

**Issue by idomic** [mlflow/mlflow#7676]
### Willingness to contribute

Yes. I can contribute this feature independently.

### Proposal Summary

I was thinking of writing some blog/docs around mlflow and [sklearn-evaluation](https://github.com/ploomber/sklearn-evaluation).

This will allow an easy mechanism for users who need to plot their experiments or evaluate their models.

### Motivation

> #### What is the use case for this feature?
Comparing multiple experiments

> #### Why is this use case valuable to support for MLflow users in general?
Saves lines of code.

> #### Why is this use case valuable to support for your project(s) or organization?
We often compare models after running multiple experiments.

> #### Why is it currently difficult to achieve this use case?
It requires moving to a notebook, and sometimes even writing code around it.

### What component(s) does this bug affect?

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [X] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [X] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [X] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [X] `area/tracking`: Tracking Service, tracking client APIs, autologging

### What interface(s) does this bug affect?

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

### What language(s) does this bug affect?

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

### What integration(s) does this bug affect?

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

**Comment by harupy** [comment]
@idomic Thanks for the FR! Could you elaborate on what the integration provides? One idea that came across my mind was to extend `mlflow.log_figure` to supports logging plots generated by `sklearn-evaluation`.

```python
from sklearn_evaluation import ...

plot = create_plot_using_sklearn_evaluation(...)
mlflow.log_figure(plot)
```

**Comment by idomic** [comment]
Yes that's a great example, sklearn-evaluation outputs matplotlib plots so that could be pretty easy to achieve (if not supported already).

Another use case I thought about is maybe a tutorial or a shared blog where we take the user from data to multiple experiments/models, and then evaluate the model's performance, for instance, the [classification example](https://sklearn-evaluation.ploomber.io/en/latest/quickstart/classification.html), we can check feature importance, maybe generate a model report and compare different confusion matrices?

**Comment by B-Step62** [comment]
A sklearn-evaluation integration is better served by a blog post or the partner's own docs than by core MLflow, and it hasn't gained traction here. Closing as out of scope, feel free to reopen with a concrete in-repo integration proposal.
